### PR TITLE
ELEMENTS-1030: add Storysource addon to Storybook

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

File: storybook/.storybook/analysis.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { knob } from '@storybook/addon-knobs';
 
-import ui from '@nuxeo/nuxeo-ui-elements/analysis.json';
+import ui from '../../ui/analysis.json';
```
Comment (nmpcunha) [inline storybook/.storybook/analysis.js]: Why changing to relative path?

File: storybook/.storybook/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    test: /\.stories\.js?$/,
+    loaders: [require.resolve('@storybook/addon-storysource/loader')],
+    enforce: 'pre',
+    include: path.resolve('../'),
+  });
+
+  return config;
+});
+
+// module.exports = function({ config }) {
```
Comment (nmpcunha) [inline storybook/.storybook/webpack.config.js]: We should avoid to commented code.

Comment (semisse) [reply storybook/.storybook/webpack.config.js]: indeed

File: storybook/.storybook/analysis.js (hunk 1 above)
Comment (semisse) [reply storybook/.storybook/analysis.js]: actually we don't need to do it, I'm going to revert it.

File: storybook/.storybook/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var path = require('path');
+const storysource = (module.exports = function({ config }) {
```
Comment (nmpcunha) [inline storybook/.storybook/webpack.config.js]: Why using a variable?

Comment (semisse) [reply storybook/.storybook/webpack.config.js]: Probably just a left over from some testing I did. Going to drop it.

File: storybook/.storybook/webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+var path = require('path');
```
Comment (Gabez0r) [inline storybook/.storybook/webpack.config.js]: Prefer `const` to `var`:
```suggestion
const path = require('path');
```

nit: no need to require the whole `path` lib if we're only using `resolve`:
```suggestion
const { resolve } = require('path');
```

and then on line 7: `include: resolve('../'),`

Comment (semisse) [reply storybook/.storybook/webpack.config.js]: Will do

**Pull request merged**